packages/core/src/common/message-service-protocol.ts

```diff
@@ -94,7 +91,7 @@ export class MessageClient {
      * To be implemented by an extension, e.g. by the messages extension.
      */
     showProgress(progressId: string, message: ProgressMessage, cancellationToken: CancellationToken): Promise<string | undefined> {
-        this.logger.info(message.text);
+        console.info(message.text);
```

Comment:
why ?

Reply:
all console calls are implemented by logger already, there is no need to inject `Logger`